pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -421,12 +421,21 @@ stdenv.mkDerivation ((drvAttrs config stdenv.hostPlatform.linux-kernel kernelPat
 
   enableParallelBuilding = true;
 
-  hardeningDisable = [ "bindnow" "format" "fortify" "stackprotector" "pic" "pie" ];
+  # These hardening features don't do any good for the kernel. The kernel takes care of its own
+  # hardening.
+  hardeningDisable = [ "all" ];
```

Comment:
Is this change still necessary, now that unwrapped LLD is being used?  If not, I'd like it to be a separate PR.

Reply:
It's not necessary. I'll split it into another PR.

Reply:
Removed.